apiserver/allfacades.go

```diff
@@ -146,7 +146,8 @@ func AllFacades() *facade.Registry {
 	reg("Backups", 1, backups.NewFacade)
 	reg("Backups", 2, backups.NewFacadeV2)
 	reg("Block", 2, block.NewAPI)
-	reg("Bundle", 1, bundle.NewFacade)
+	reg("Bundle", 1, bundle.NewFacadeV1)
+	reg("Bundle", 2, bundle.NewFacadeV2)
```

Comment:
Same here. We will have V2 available for testing, but until we have the implementation finished, we don't actually want to expose it over the API.